cypress/e2e/checkin/check-in-claimed-returned-items.cy.js

```diff
@@ -0,0 +1,199 @@
+  };
+  const itemsData = {
+    itemsWithSeparateInstance: [
+      { title: `InstanceForFoundByLibrary ${getRandomPostfix()}` },
+      { title: `InstanceForReturnedByPatron ${getRandomPostfix()}` },
+    ],
+  };
+
+  before('Preconditions', () => {
+    itemsData.itemsWithSeparateInstance.forEach(function (item, index) {
+      item.barcode = generateUniqueItemBarcodeWithShift(index);
+    });
+
+    cy.getAdminToken()
+      .then(() => {
+        ServicePoints.createViaApi(testData.servicePointS);
+        testData.defaultLocation = Location.getDefaultLocation(testData.servicePointS.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+          testData.materialType = materialTypes.name;
+        });
+      })
+      .then(() => {
+        itemsData.itemsWithSeparateInstance.forEach((item, index) => {
+          InventoryInstances.createFolioInstanceViaApi({
+            instance: {
+              instanceTypeId: testData.instanceTypeId,
+              title: item.title,
+            },
+            holdings: [
+              {
+                holdingsTypeId: testData.holdingTypeId,
+                permanentLocationId: testData.defaultLocation.id,
+              },
+            ],
+            items: [
+              {
+                barcode: item.barcode,
+                status: { name: 'Available' },
+                permanentLoanType: { id: testData.loanTypeId },
+                materialType: { id: testData.materialTypeId },
+              },
+            ],
+          }).then((specialInstanceIds) => {
+            itemsData.itemsWithSeparateInstance[index].instanceId = specialInstanceIds.instanceId;
+            itemsData.itemsWithSeparateInstance[index].holdingId = specialInstanceIds.holdingIds[0].id;
+            itemsData.itemsWithSeparateInstance[index].itemId = specialInstanceIds.holdingIds[0].itemIds;
+            itemsData.itemsWithSeparateInstance[index].materialType = testData.materialType;
+          });
+        });
+        cy.wrap(itemsData.itemsWithSeparateInstance).as('items');
+      });
+
+    PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+      patronGroup.id = patronGroupResponse;
+    });
+
+    cy.createTempUser(
+      [
+        permissions.checkinAll.gui,
+        permissions.loansAll.gui,
+        permissions.uiUsersLoansClaimReturned.gui,
+        permissions.uiUsersView.gui,
```

Comment:
Do you need all these permissions?
in the test case there are two permissions:
1)Check in: All permissions
2)Users: User loans view

Reply:
removed